yarn.lock

```diff
@@ -2627,6 +2627,16 @@ browserify-zlib@^0.2.0:
   dependencies:
     pako "~1.0.5"
 
+<<<<<<< HEAD
+browserslist@4.7.0, browserslist@^4.0.0, browserslist@^4.1.1, browserslist@^4.4.2, browserslist@^4.6.0, browserslist@^4.7.0:
```

Comment:
Make sure you're always recreating lockfiles after pulling in master

Reply:
Done